src/components/CodeHighlight/CodeHighlight.js

```diff
@@ -0,0 +1,61 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+
+import * as Languages from './Languages';
+import * as Themes from './Themes';
+
+const { LANGUAGES } = Languages;
+const {
+  THEMES,
+  themes,
+} = Themes;
+
+const markup = val => {
+  return { __html: val };
+};
+
+export const CodeHighlight = memo(props => {
+  const {
+    code,
+    language,
+    theme,
+    usePre,
+    className,
+  } = props;
+
+  const hlTheme = themes[theme];
```

Comment:
Not sure I understand what `hlTheme` means. Please give a more reasonable name.

Reply:
Done